beacon-chain/cache/depositcache/pending_deposits_test.go

```diff
@@ -90,7 +90,7 @@ func TestPendingDeposits_OK(t *testing.T) {
 		{Proof: [][]byte{[]byte("A")}},
 		{Proof: [][]byte{[]byte("B")}},
 	}
-	assert.DeepEqual(t, expected, deposits)
+	assert.DeepSSZEqual(t, expected, deposits)
```

Comment:
why is this changed ?

Reply:
Reflect no longer works for v2 protobuf structs because of unexported fields. And this is an array of proto messages, so the only equality check the works  really is DeepSSZEqual.